scripts/50-clone-epiphany-examples-and-parallella-examples.sh

```diff
@@ -4,5 +4,5 @@ set -o pipefail
 set -u
 
 cd mnt/rootfs/home/parallella
-git clone https://github.com/adapteva/epiphany-examples.git --branch 2016.11
-git clone https://github.com/parallella/parallella-examples.git --branch 2016.11
+git clone --depth=1 https://github.com/adapteva/epiphany-examples.git --branch 2016.11
```

Comment:
No to these two. We might need to ask users to switch branches etc. to troubleshoot things etc.